javav2/example_code/pinpoint/src/main/java/com/example/pinpoint/GetTemplateByName

```diff
@@ -0,0 +1,39 @@
+package com.notification.poc;
+/* Created by Seemant Shukla on 25/10/21 */
```

Comment:
Delete this line. Once you submit a PR, Amazon owns the code.

Reply:
I talked to our Java code writer. He wants to give you credit for your submissions, so we'll add some metatdata to your submissions like:

//snippet-sourceauthor:[Seemant Shukla]